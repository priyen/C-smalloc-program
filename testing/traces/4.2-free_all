m 0 16
m 1 16
m 2 16
m 3 16
m 4 16
m 5 16
m 6 16
m 7 16
m 8 16
m 9 16
m 10 16
m 11 16
m 12 16
m 13 16
m 14 16
m 15 16
m 16 16
m 17 16
m 18 16
m 19 16
m 20 16
f 3
f 13
f 2
f 12
f 6
f 18
f 0
f 10
f 14
f 9
f 17
f 5
f 1
f 7
f 19
f 15
f 4
f 8 
f 16
f 20
f 11
