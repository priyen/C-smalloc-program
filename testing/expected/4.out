Total free mem: 261808
Total allocated mem: 336
Number of allocated blocks: 21
